apps/examples/src/index.tsx

```diff
@@ -43,7 +44,11 @@ const router = createBrowserRouter([
 	},
 	{
 		path: 'end-to-end',
-		element: <EndToEnd />,
+		lazy: async () => ({ element: <EndToEnd /> }),
```

Comment:
fix: this wasn't doing lazy loading like the other examples. this was causing some styling issues